Dockerfile

```diff
@@ -41,5 +43,4 @@ FROM base AS final
 
 WORKDIR /tmp
 
-ENTRYPOINT ["/usr/src/cloc"]
-CMD ["--help"]
+CMD ["/usr/src/cloc", "--help"]
```

Comment:
Allowing to run cloc interactively, which otherwise is not practical with Docker and some of the examples given in the README.md.